backend/OpenAPI.yaml

```diff
@@ -126,27 +233,30 @@ paths:
                 description:
                   type: string
                   example: "Lightly used gaming laptop, great condition!"
-                seller_id:
-                  type: integer
-                  example: 101
       responses:
         "201":
           description: Listing created successfully
+        "500":
+          description: Internal Server Error
           content:
             application/json:
               schema:
                 type: object
                 properties:
-                  id:
-                    type: integer
-                    example: 10
-                  title:
+                  error_code:
                     type: string
-                    example: "Gaming Laptop for sale"
-                  price:
-                    type: number
-                    format: float
-                    example: 899.99
-                  description:
+                    example: "INTERNAL_SERVER_ERROR"
+                  message:
+                    type: string
+                    example: "An unexpected error occurred."
+                  details:
                     type: string
```

Comment:
Overall, I think it's best that we come together as a backend sub-team to agree on an error response schema.